### PR TITLE
Added Chase Corbitt to guestbook

### PR DESCRIPTION
**Pull request by metaltheory**
## What type of PR is this? (check all applicable)

- [x] 🤝 Add a contributor
- [ ] 📝 Documentation Update

## Contributors checklist (check all applicable)

- Have you read the [Getting Started](https://github.com/open-sauced/guestbook#getting-started) section thoroughly?

  - [x] ✅ Yes
  - [ ] ❌ Not yet

- How did you add yourself as a contributor?

  - [ ] 🤖 With CLI
  - [x] ⌨️ Manually

    If you added yourself manually, did you follow the [emoji key and contribution types](https://allcontributors.org/docs/en/emoji-key) to fill in the value?

    - [x] ✅ Yes
    - [ ] ❌ No

- Have you run `npm run contributors:generate` to generate your profile and the badge on the README?
  - [x] ✅ Yes
  - [ ] ❌ No

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by adiati98** [comment]
Hey @metaltheory,
Thank you for your PR!

There are a couple of things we'd like you to do:

- Please fill all the areas in the PR form so we can review your PR.
- We have merged a PR before yours. Please [resolve the conflicts](https://github.com/open-sauced/guestbook#keeping-branch-up-to-date-and-resolving-merge-conflicts) before we can merge yours.

Let us know if you need any help 😊

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @metaltheory,
Thank you for your PR!

There are a couple of things we'd like you to do:

- Please fill all the areas in the PR form so we can review your PR.
- We have merged a PR before yours. Please [resolve the conflicts](https://github.com/open-sauced/guestbook#keeping-branch-up-to-date-and-resolving-merge-conflicts) before we can merge yours.

Let us know if you need any help 😊

**Comment by adiati98** [comment]
Hi @metaltheory,

Have you run the `npm run contributors:generate` command?

File: README.md [hunk 1]
```diff
@@ -208,6 +208,7 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-99-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-98-orange.svg?style=flat-square)](#contributors-)
```
Comment (adiati98) [inline README.md:211]: This is duplicate with line 210.
Please remove one, then run the `npm run contributors:generate` command in your terminal.

**Review by adiati98 — CHANGES_REQUESTED** [review]

**Comment by metaltheory** [comment]
Okay- removed the duplicate line at 211. Let me know if there are any other changes I need to make. Thanks!

**Comment by adiati98** [comment]
> Okay- removed the duplicate line at 211. Let me know if there are any other changes I need to make. Thanks!

Have you run the `npm run contributors:generate` command in your terminal?

**Comment by metaltheory** [comment]
I think I failed to do that after resolving conflicts - I can the command and pushed. Hope it's okay now -  fingers crossed! 🤞

**Review by adiati98 — APPROVED** [review]
Well done, @metaltheory! 🙌
Thank you for the changes! 😄

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕 

git merge GIF

**Pull request merged**